parsers/lib/quality.py

```diff
@@ -41,6 +42,18 @@ def validate_exchange(item, k):
         raise ValidationError('datetime %s is not valid for %s' %
                               (item['datetime'], k))
     validate_reasonable_time(item, k)
+    # Verify that the exchange flow is not greater than the interconnector capacity
+    # Use https://github.com/tmrowco/electricitymap-contrib/blob/master/parsers/example.py for expected format
+    if item.get('sortedZoneKeys', None) and item.get('netFlow', None):
+        zone_names = item['sortedZoneKeys']
+        if len(zone_names) == 2:
+                if ((zone_names in EXCHANGES_CONFIG) and
+                    ('capacity' in EXCHANGES_CONFIG[zone_names])
+                ):
+                    interconnector_capacities = EXCHANGES_CONFIG[zone_names]['capacity']
+                    if not (min(interconnector_capacities) <= item['netFlow'] <= max(interconnector_capacities)):
```

Comment:
Capacity data might be outdated or imprecise. I would suggest adding a margin of error (10%?). What do you think?

Reply:
Good idea indeed, but is there to way to evaluate what's the "expected" level of discrepancy between what data we have and the truth?

Reply:
I'm not sure to be honest. We could start with 10% and see if any exchange fail?

Reply:
>Capacity data might be outdated or imprecise. I would suggest adding a margin of error (10%?). What do you think?

Note that the number might also be inflated due to transmission losses, especially over long distances